tokio/src/task/blocking.rs

```diff
@@ -37,12 +37,11 @@ cfg_rt_threaded! {
 }
 
 cfg_blocking! {
-    /// Runs the provided closure on a thread where blocking is acceptable.
+    /// Runs the provided closue on a thread where blocking is acceptable 
```

Comment:
If we fix the typo here, I think this is ready to merge :)

Reply:
lol!
I don't know why I am making so many mistakes. Sorry for the troubles.